website/docs/docs/dbt-versions/release-notes/02-Nov-2023/job-notifications-rn.md

```diff
@@ -8,8 +8,10 @@ tags: [Nov-2023]
 
 There are new quality-of-life improvements in dbt Cloud for email and Slack notifications about your jobs: 
 
-- You can add external email addresses and receive job notifications from them. External emails can be addresses that are outside of your dbt Cloud account and also for configuring notifications in third-party services like Microsoft Teams and PagerDuty. 
+- You can add external email addresses and receive job notifications from them. External emails can be:
```

Comment:
Should this be "and send job notifications to them"?

Reply:
oh that's weird, i totally remember updating it. second try